src/tools/rest-backend/README.md

```diff
@@ -56,6 +56,31 @@ The REST service can also be configured. The file `rest-backend-config.js` in th
 
 To stop the service, run `sh stop-rest-backend.sh` in the same path.
 
+### Configure as service
+
+To configure the rest-backend as service, it is possible to use `systemd` on most systems.
+
+1) Create a new service file under `/etc/systemd/system`, e.g. `touch /etc/systemd/system/rest-backend.service`.
```

Comment:
Why first creating the file and then filling it with content later? What about using `cat` instead:
```
cat > /etc/systemd/system/rest-backend.service << HERE
[Unit]
...
HERE
```

Or do you want to explain that the folder `/etc/systemd/system` needs to be created first?